Os/ValidateFileCommon.cpp

```diff
@@ -18,10 +18,10 @@ namespace Os {
 
         // Get the file size:
         FileSystem::Status fs_status;
-        FwSizeType fileSize = 0;
+        FwSignedSizeType fileSize = 0;
```

Comment:
Why are we switching to signed size?

Reply:
The File API uses signed size, and these values are passed in and out of that API.